report/MVP/MVP.tex

```diff
@@ -0,0 +1,23 @@
+This chapter will talk about what our Minimum Viable Product (MVP) is, and we will in the following chapters go though how we have developed this MVP.
+
+\section{What is our MVP?}
+An MVP is a piece of product, that gives some sort of value, with the least amount of work went into it.
```

Comment:
```suggestion
A MVP is a piece of product, that gives some sort of value, with the least amount of work went into it.
```

Reply:
Men er det korrekt?
Det kan vel også bare være den del af en løsning en kunde kan give feedback på. Så tidligt som muligt.

Reply:
I følge den kilde jeg fandt, så er en MVP et stykke produkt, som giver en form for value eller validate, som kræver det mindst muligt arbejde.